### PR TITLE
CORE-52: Add -f flag to start in foreground

### PR DESCRIPTION
**Pull request by simonl2002**

### CODE REVIEW

File: src/astmanproxy.c [hunk 1]
```diff
@@ -720,8 +725,8 @@ int main(int argc, char *argv[])
 		exit(1);
 	}
 
-	/* If we are not in debug mode, then fork to background */
-	if (!debug) {
+	/* If we are not in foreground mode, then fork to background */
+	if (!foreground) {
```
Comment (tommymei) [inline src/astmanproxy.c:721]: @simonl2002 Question:
1. So we won't get any logging now with foreground mode, intentional?
2. If we ever need to run it in debug mode, it'll run in background and the logs won't go to the console.

Comment (simonl2002) [reply src/astmanproxy.c:721]: yeah we'd have to run with -f -d

**Review by tommymei — APPROVED** [review]
looks good

**Comment by tommymei** [comment]
looks good

**Review by tommymei — APPROVED** [review]
looks good

**Pull request merged**